### ISSUE TITLE
Image picker cant find images

### ISSUE THREAD

**Issue by iaK** [JimBobSquarePants/UmbracoFileSystemProviders.Azure#94]
I'm trying to import images into the rich text editor (Umbraco.TinyMCEv3) using the media picker. After i installed this plugin, no images show up in the media picker overlay. All the folders show up correctly. Any ideas?

Umbraco version 7.5.14.

**Comment by JimBobSquarePants** [comment]
Hi @iaK 

Can you see any media in the media section?

Have you ensured the Virtual Path Provider configuration is set up correctly? That throws quite a few people.

Cheers

James

**Comment by iaK** [comment]
Hi! Thanks for answering.

All the media show up in the media section as expected, no problem there.

I've removed this chunk from root web.config

```
    <location path="Media">
      <system.webServer>
        <handlers>
          <remove name="StaticFileHandler" />
          <add name="StaticFileHandler" path="*" verb="*" preCondition="integratedMode" type="System.Web.StaticFileHandler" />
        </handlers>
      </system.webServer>
    </location>
```

and added this chunk in the /media/web.config file 

```
<?xml version="1.0" encoding="UTF-8"?>
<configuration>
	<system.webServer>
		<handlers>
			<clear />
			<add name="StaticFileHandler" path="*" verb="*" preCondition="integratedMode" type="System.Web.StaticFileHandler" />
			<add name="StaticFile" path="*" verb="*" modules="StaticFileModule,DefaultDocumentModule,DirectoryListingModule" resourceType="Either" requireAccess="Read" />
		</handlers>
	</system.webServer>
</configuration>
```

But still experiencing the same problem..

**Comment by beriniwlew** [comment]
@iaK Did you ever figure out what it was?

**Comment by iaK** [comment]
Nope, never did. I got put off that project and have no idea how they solved it unfortunately. :/

**Comment by beriniwlew** [comment]
Looking at the JavaScript Console, it says "Cannot resolve the file url from the mediaEntity, it does not contain the required metaData" when I go to a document with my media pickers.

**Comment by Rushabhmaster** [comment]
Hey Guys I'm having the same problem.
**Media section is fine** but can not see images on pickers.
![image](https://user-images.githubusercontent.com/29447542/76896097-1ca46200-68f6-11ea-857e-5c6d54176d4f.png)
![image](https://user-images.githubusercontent.com/29447542/76896188-3e054e00-68f6-11ea-8477-d0d0e75298f3.png)
